pkgs/development/ocaml-modules/fix/default.nix

```diff
@@ -2,12 +2,13 @@
 
 assert lib.versionAtLeast (lib.getVersion ocaml) "3.12";
 
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
 
-  name = "ocaml-fix-20130611";
+  pname = "fix";
+  version = "20201120";
 
   src = fetchurl {
-    url = "http://gallium.inria.fr/~fpottier/fix/fix-20130611.tar.gz";
+    url = "https://gitlab.inria.fr/fpottier/fix/-/archive/${version}/archive.tar.gz";
     sha256 = "1phlqcs1nb93x9cf0w0hnq2ck4dmn71zm4mxf60w96vb9yb9qzp0";
```

Comment:
The hash is wrong.

Reply:
Unforgivable. Sorry. I'm still trying to understand how this happened. O.o

Reply:
Now using fetchFromGitLab, hope it is fine.